.github/workflows/codeql-analysis.yml

```diff
@@ -1,67 +1,33 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
-#
-# ******** NOTE ********
-# We have attempted to detect the languages in your repository. Please check
-# the `language` matrix defined below to confirm you have the correct set of
-# supported CodeQL languages.
-#
-name: "CodeQL"
+name: "Code Scanning - Action"
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    # The branches below must be a subset of the branches above
-    branches: [ master ]
+    branches: [master]
   schedule:
-    - cron: '43 23 * * 1'
+    - cron: '30 1 * * 0'
 
 jobs:
-  analyze:
-    name: Analyze
+  CodeQL-Build:
```

Comment:
Keeping a job name in `lower_camel_case` is more standard, and "analyze" is the default name provided by CodeQL since it can be used directly by GitHub (since this Workflow is GitHub-managed).

To change the displayed job name, use the `jobs.<id>.name` property